src/Sylius/Bundle/AttributeBundle/Form/EventSubscriber/BuildAttributeValueFormSubscriber.php

```diff
@@ -29,11 +29,17 @@ class BuildAttributeValueFormSubscriber implements EventSubscriberInterface
     protected $attributeRepository;
 
     /**
+     * @var string
+     */
+    protected $subjectName;
+
+    /**
      * @param RepositoryInterface $attributeRepository
      */
-    public function __construct(RepositoryInterface $attributeRepository)
+    public function __construct(RepositoryInterface $attributeRepository, $subject = null)
     {
         $this->attributeRepository = $attributeRepository;
+        $this->subjectName = $subject;
```

Comment:
Do we have any convention here? Nevertheless use subject or subjectName, no both